docs/_docs/snapshots/snapshots.adoc

```diff
@@ -146,29 +146,23 @@ See the link:tools/control-script#checking-snapshot-consistency[Control Script]
 commands.
 
 == Restoring From Snapshot
-
-Currently, the data restore procedure has to be performed manually. In a nutshell, you need to stop the cluster,
-replace persistence data and other files with the data from the snapshot, and restart the nodes.
+=== Manual snapshot restore procedure
```

Comment:
Having 2 headers in a row is generally bad for readability. Please add a small section between them to explain whats in the section.

Additionally, headers should be capitalised.

Manual Snapshot Restore Procedure

Reply:
Done, thanks

Reply:
I have added a short description of both procedures after the heading: 
``The snapshot can be restored manually on a stopped cluster or automatically on an active cluster. Both procedures are described below.``
is it ok?

Reply:
Yes, this works.